lib/msf/core/post/linux/kernel.rb

```diff
@@ -0,0 +1,58 @@
+# -*- coding: binary -*-
+require 'msf/core/post/common'
+
+module Msf
+class Post
+module Linux
+module Kernel
+  include ::Msf::Post::Common
+
+  #
+  # Returns the kernel version
+  #
+  # @return [String]
+  #
+  def kernel_version
+    cmd_exec('uname -r').to_s
```

Comment:
Should this be returned as a Gem::Version? easier to do > and < operations on, harder to do a regex on... pros and cons here.

Reply:
Returning the raw string is preferable. `Gem::Version` has trouble with `-`